services/ui-src/src/components/fields/ChoiceListField.test.tsx

```diff
@@ -224,10 +227,13 @@ describe("<ChoiceListField />", () => {
 
     test("CheckboxField should render nested child fields for choices with children", async () => {
       // Render Initial State and choices
+      mockGetValues(undefined);
```

Comment:
Fixes `Warning: Failed prop type: Invalid prop valuesupplied toTextInput, expected one of type [string, number].`.